pjsip/src/pjsip/sip_parser.c

```diff
@@ -1905,8 +1905,8 @@ static void int_parse_contact_param( pjsip_contact_hdr *hdr,
                 hdr->expires--;
             if (hdr->expires > PJSIP_MAX_EXPIRES)
                 hdr->expires = PJSIP_MAX_EXPIRES;
-            if (hdr->expires < PJSIP_MIN_EXPIRES)
-                hdr->expires = PJSIP_MIN_EXPIRES;
+            // if (hdr->expires < PJSIP_MIN_EXPIRES)
+            //     hdr->expires = PJSIP_MIN_EXPIRES;
```

Comment:
The check may be needed when `PJSIP_MIN_EXPIRES` is not zero (not sure if it is possible though)? So what if instead of commenting it out, change it to something like
```
#if PJSIP_MIN_EXPIRES != 0
   if (hdr->expires < PJSIP_MIN_EXPIRES)
       hdr->expires = PJSIP_MIN_EXPIRES;
#endif
```